modules/channel-web/src/views/lite/components/messages/MessageList.tsx

```diff
@@ -98,6 +98,7 @@ class MessageList extends React.Component<MessageListProps> {
         groups.push(currentGroup)
       }
 
+      if (currentGroup.find(x => x.id === m.id)) return
```

Comment:
What is this line used for ? Also, brackets are required, even for a simple return. It keeps the code more clear.

Reply:
It fix for bug, that comes from new re-rendering of IntlProvider root element. If we could remove this line, we could get messages display multiplication after locale change.